dev/com.ibm.ws.install/src/com/ibm/ws/install/maven/MavenDirectoryInstall.java

```diff
@@ -0,0 +1,313 @@
+/*******************************************************************************n * Copyright (c) 2019 IBM Corporation and others.n * All rights reserved. This program and the accompanying materialsn * are made available under the terms of the Eclipse Public License v1.0n * which accompanies this distribution, and is available atn * http://www.eclipse.org/legal/epl-v10.htmln *n * Contributors:n *     IBM Corporation - initial API and implementationn *******************************************************************************/
+package com.ibm.ws.install.maven;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import com.ibm.ws.install.InstallException;
+import com.ibm.ws.install.internal.InstallKernelMap;
+import com.ibm.ws.install.internal.InstallLogUtils;
+import com.ibm.ws.install.internal.InstallLogUtils.Messages;
+import com.ibm.ws.kernel.boot.cmdline.Utils;
+
+/**
+ *
+ */
+public class MavenDirectoryInstall {
+
+    private final InstallKernelMap map;
+    private final File fromDir;
+    private String toExtension;
+    private String openLibertyVersion;
+    private Logger logger;
+    private boolean tempCleanupRequired;
+
+    private final String TEMP_DIRECTORY =  Utils.getInstallDir() + "/tmp/";
+    private final String OPEN_LIBERTY_PRODUCT_ID = "io.openliberty";
```

Comment:
Please remove OPEN_LIBERTY_PRODUCT_ID , OPEN_LIBERTY_GROUP_ID , and CLOSED_LIBERTY_GROUP_ID  since they are not used.